junit-common/src/main/java/org/jboss/weld/junit/common/WeldCDIExtension.java

```diff
@@ -29,16 +29,17 @@
 /**
  *
  * @author Martin Kouba
+ * @author <a href="mailto:manovotn@redhat.com">Matej Novotny</a>
  */
-class WeldJunit4Extension implements Extension {
+public class WeldCDIExtension implements Extension {
```

Comment:
The class, construct and methods may stay package-private - methods are only invoked by `AbstractWeldInitiator` and the class itself is accessed by `MockBean` which should only

Reply:
Well, if we settle on the `MockBean`, then yes. I had it like this just to play around with it, I suppose this might appear on multiple places.